src/modules/ekf2/EKF/python/tuning_tools/mc_wind_estimator/readme.md

```diff
@@ -27,6 +32,8 @@ The estimated parameters are displayed in the console and the fit quality is sho
 param set EKF2_BCOEF_X 0.0
 param set EKF2_BCOEF_Y 62.1
 param set EKF2_MCOEF 0.16
-/!\EXPERIMENTAL param set EKF2_DRAG_NOISE 0.31
```

Comment:
That's just the raw output of the script in the console. It's printed like that (without newline) because I want to force the user to remove the `/!\EXPERIMENTAL` in front of the `param set` if he really wants to use that value.